index.js

```diff
@@ -1,13 +1,19 @@
 #!/usr/bin/env node
 const open = require('open-pip')
 const ora = require('ora')
+const getStdin = require('get-stdin')
 const spinner = ora().start()
 
-const input = process.argv[2]
+getStdin()
+.then(stdin => {
+	if (stdin) return stdin.trim();
```

Comment:
remove semicolon 👌 


I'll set up the linter to run in travis when i get home 🎉